CHANGELOG.md

```diff
@@ -7,8 +7,11 @@
 
 *Enchantments*
 - The KubernetesDeploy::RenderTask now supports a template_paths argument ([#555](https://github.com/Shopify/kubernetes-deploy/pull/546))
+- Validate secrets with `--server-dry-run` on supported clusters. ([#553](https://github.com/Shopify/kubernetes-deploy/pull/553))
 - We no longer hide errors from apply if all sensitive resources have passed server-dry-run validation. ([#570](https://github.com/Shopify/kubernetes-deploy/pull/570))
-
+- **[Breaking change]** Added PersistentVolumeClaim to the prune whitelist. ([#573](https://github.com/Shopify/kubernetes-deploy/pull/573))
```

Comment:
0.29.0 was already released. New stuff should be going under "next" above.

Reply:
Ah, OK, I was confused by this [compare](https://github.com/Shopify/kubernetes-deploy/compare/v0.29.0...master) .
I'll put it in `next` section